octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2

```diff
@@ -1,25 +1,13 @@
 
 {% if language == 'de' %}
-
     <script>var wizard_plugin_corewizard_whatsnew_0_title_de = "Was gibt's neues";</script>
```

Comment:
This variable (wizard_plugin_corewizard_whatsnew_0_title_de) should be removed. also from the ViewModel.

Reply:
ViewModel?